src/prefect/core/flow.py

```diff
@@ -568,30 +568,54 @@ def chain(self, *tasks: Task, validate: bool = None) -> List[Edge]:
             )
         return edges
 
-    def update(self, flow: "Flow", validate: bool = None) -> None:
+    def update(
+        self, flow: "Flow", parmerge: bool = None, validate: bool = None
+    ) -> None:
         """
-        Take all tasks and edges in another flow and add it to this flow
-
+        Take all tasks and edges in another flow and add it to this flow.  
+        When `parmerge` is set to`True` -- Duplicate parameters in the unput `flow`
+        are replaced with those in the flow being updated. 
+    
         Args:
-            - flow (Flow): A flow which is used to update this flow
-            - validate (bool, optional): Whether or not to check the validity of the flow
-
+            - flow (Flow): A flow which is used to update this flow.
+            - parmerge (bool, optional): Merge two graphs on parameters
+            - validate (bool, optional): Whether or not to check the validity of the flow.
+        
         Returns:
             - None
         """
+
         for task in flow.tasks:
-            if task not in self.tasks:
+
+            duped_task = self.get_tasks(slug=task.slug) if parmerge else list()
```

Comment:
I see what you're getting at here but I think we should make the code more explicit that this is solely for Parameters, so something like:
```python
if merge_parameters and isinstance(task, Parameter):
    duped_params = [p for p in self.parameters() if p.name == task.name]
else:
    duped_params = []
```